tpm2/tpm2.go

```diff
@@ -250,6 +250,24 @@ func encodePasswordAuthArea(passwords ...string) ([]byte, error) {
 	return concat(size, res)
 }
 
+func encodeAuthArea(sessionHandle tpmutil.Handle, password string) ([]byte, error) {
```

Comment:
This looks very similar to `encodePasswordAuthArea`.
How about changing `encodePasswordAuthArea` to
```
func encodePasswordAuthArea(password string) ([]byte, error) {
  return encodeAuthArea(HandlePasswordSession, password)
}
```

Reply:
encodePasswordAuthArea takes multiple passwords, so they aren't quite the same...would take more of a refactor to merge the two

Reply:
Can you change this to take multiple passwords perhaps?
It can look identical to current encodePasswordAuthArea minus passing through sessionHandle

Reply:
done